common/source/docs/common-buttons.rst

```diff
@@ -37,16 +39,16 @@ The pin is automatically configured with an internal pullup meaning the button o
 PWM Input Setup
 ---------------
 
-A PWM input can be configured as a button input.  The parameter changes listed below are for the 1st button but the 2nd, 3rd and 4th buttons can be setup similarly.
+PWM signals can be used as button inputs.  PWM values >1800us or <1200us can be used to trigger the Auxiliary Function. The parameter changes listed below are for the 1st button, but the 2nd, 3rd and 4th buttons can be setup similarly. As above, to determine which GPIO pin number corresponds to which autopilot SERVO/MOTOR output pin or dedicated GPIO pin, the autopilot's hwdef.dat file must be consulted. See :ref:`common-gpios` for how to determine this.
 
-- Set :ref:`BTN_PIN1<BTN_PIN1>` to the number of the pin connected to the button/switch (e.g. "55" = "AUXOUT6")
-- Set :ref:`BTN_OPTIONS1<BTN_OPTIONS1>` = to "1" to trigger the auxiliary function with a PWM > 1800us
-- Set :ref:`BTN_OPTIONS1<BTN_OPTIONS1>` = to "3" to trigger the auxiliary function with a PWM < 1200us
-- Set :ref:`BTN_FUNC1<BTN_FUNC1>` to the :ref:`Auxiliary Function <common-auxiliary-functions>` to be triggered.  If set to 0, no function is triggered
+- Set :ref:`BTN_PIN1<BTN_PIN1>` to the number of the GPIO connected to the button/switch (e.g. "55" = "AUXOUT6")
+- Set :ref:`BTN_OPTIONS1<BTN_OPTIONS1>` = to "1" to be interpreted as a "HIGH" position by the auxiliary function with a PWM > 1800us
+- Set :ref:`BTN_OPTIONS1<BTN_OPTIONS1>` = to "3" to be interpreted as a "LOW" position by the auxiliary function with a PWM < 1200us
```

Comment:
Ah, I think this change is incorrect.  With BTN_OPTIONS1 = 3 the button's HIGH position is triggered when the PWM < 1200us.

Reply:
right...